Started by timer
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building in workspace /var/jenkins_home/workspace/fdc-prod/SNAA-Daily-Issue-Report
The recommended git tool is: NONE
using credential vinay-github
 > git rev-parse --resolve-git-dir /var/jenkins_home/workspace/fdc-prod/SNAA-Daily-Issue-Report/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/The-Frontdoor-Collective/fdc-etl.git # timeout=10
Fetching upstream changes from https://github.com/The-Frontdoor-Collective/fdc-etl.git
 > git --version # timeout=10
 > git --version # 'git version 2.30.2'
using GIT_ASKPASS to set credentials vinay-github
 > git fetch --tags --force --progress -- https://github.com/The-Frontdoor-Collective/fdc-etl.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/develop^{commit} # timeout=10
Checking out Revision a1eef3b1fb2da8c370ffbc149fb44d7214209c7a (refs/remotes/origin/develop)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f a1eef3b1fb2da8c370ffbc149fb44d7214209c7a # timeout=10
Commit message: "Merge pull request #20 from The-Frontdoor-Collective/ITDMF-126"
 > git rev-list --no-walk a1eef3b1fb2da8c370ffbc149fb44d7214209c7a # timeout=10
[SNAA-Daily-Issue-Report] $ /bin/sh -xe /tmp/jenkins8173653056119747755.sh
+ cd python_scripts
+ python3 daily_exceptions_report_by_fdp_id.py 14
Location Name :  SNAA
fdp IDs :  ['31', '33']
loop Start for fdp ID:  31
Email has been sent.
loop Start for fdp ID:  33
Traceback (most recent call last):
  File "/var/jenkins_home/workspace/fdc-prod/SNAA-Daily-Issue-Report/python_scripts/daily_exceptions_report_by_fdp_id.py", line 133, in <module>
    wb.save(workbook_name)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/openpyxl/workbook/workbook.py", line 407, in save
    save_workbook(self, filename)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/openpyxl/writer/excel.py", line 293, in save_workbook
    writer.save()
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/openpyxl/writer/excel.py", line 275, in save
    self.write_data()
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/openpyxl/writer/excel.py", line 89, in write_data
    archive.writestr(ARC_WORKBOOK, writer.write())
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/openpyxl/workbook/_writer.py", line 148, in write
    self.write_views()
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/openpyxl/workbook/_writer.py", line 135, in write_views
    active = get_active_sheet(self.wb)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/openpyxl/workbook/_writer.py", line 33, in get_active_sheet
    raise IndexError("At least one sheet must be visible")
IndexError: At least one sheet must be visible
Build step 'Execute shell' marked build as failure
Finished: FAILURE
